lib/client/index.js

```diff
@@ -76,7 +76,20 @@ client.init = function init (callback) {
       console.log('Already tried to get settings after auth, but failed');
     } else {
       client.settingsFailed = true;
-      language.set('en');
+
+      // detect browser language
+      var lang = Storages.localStorage.get('language') || (navigator.language || navigator.userLanguage).toLowerCase();
```

Comment:
So I changed it. Please don't forget that for the VERY FIRST authentication dialog it is better to show it in the browser language, than in english.
E.g. my browser language is german and I would like to see the auth dialog in german.